src/api/googlesheetApi.js

```diff
@@ -23,17 +26,35 @@ function normalizeHeaders(element) {
 
 }
 
+function sheetIdFromPath(directory, path) {
+  if(!!path || path === ""){
```

Comment:
I assume there is some terse es6 way to do stuff like this better. Feel free to suggest, otherwise I'll look into it tomorrow.